src/components/Header/Header.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PandaLogo from "../../assets/images/pandalogo.png";
+import ProfileImg from "../../assets/images/profile.png";
+import { Link, NavLink } from 'react-router-dom';
+import './Header.css';
+
+function header() {
+  return (
+    <header>
+      <div className="header-container">
+        <div className="header-wrap">
+          <Link to="/">
+            <img className="panda-logo" src={PandaLogo} alt="판다마켓로고" />
+          </Link>
+          <div className="nav-container">
+            <NavLink to="/community" activeClassName="active">자유게시판</NavLink>
```

Comment:
P2:
해당 아이템이 active일 때 특정 classname을 주시려면 아래처럼 하시면 됩니다!
관련 문서도 확인해보시면 좋을 것 같아요.

https://reactrouter.com/en/main/components/nav-link

```suggestion
            <NavLink to="/community"  className={({ isActive }) => (isActive ? "active" : "")}>자유게시판</NavLink>
```